spec/features/finance/npq/course_payment_breakdown_spec.rb

```diff
@@ -16,28 +16,37 @@
     Finance::NPQ::CalculationOverviewOrchestrator.call(
       cpd_lead_provider: cpd_lead_provider,
       event_type: :started,
+      interval: nil,
```

Comment:
can we make the keyword argument default to `nil` in method definition rather than having a rogue nil here?

Reply:
It is set to nil in the method, I added it here when I was trying to understand why the output specs weren't running. Arguably this could be removed

Reply:
Now updated 👍